scripts/src/main/resources/scripts/command/lazydocker

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+# Call lazydocker with specified arguments.
+function doRun() {
+  doSetup silent
+  cd "${LAZYDOCKER_HOME}" || exit 1
+  if doIsWindows
+  then
+    powershell.exe -command 'start powershell {.\lazydocker.exe}'
+  else
+    ./lazydocker 
+  fi
+}
+
+function doSetup() {
+  if [ ! -d "${LAZYDOCKER_HOME}" ] || [ "${1}" != "silent" ]
+  then
+    local version="${LAZYDOCKER_VERSION}"
+    # Get latest release
+    if [ -z "${version}" ]
+    then 
+      version="0.18.1"
+      doEcho "Getting latest release..."
+      local latestversion
+      latestversion=$(curl -Ls -o /dev/null -w '%{url_effective}' https://github.com/jesseduffield/lazydocker/releases/latest | sed 's/^.*\///' | sed 's/^v//')
+      doVersionCompare "${latestversion}" "${version}"
+      if [ "${?}" = 1 ]
+      then
+        version=${latestversion}
+      fi
+    fi
+    doCheckDockerVersions
+    doInstall "-" "${LAZYDOCKER_HOME}" "lazydocker" "${version}"
+  fi
+}
+
+function doCheckDockerVersions() {
+  local DOCKERENGVERSION
+  DOCKERENGVERSION=$(docker version | grep "Version" | head -1 | sed -ne 's/.*\([0-9][0-9]\+\.[0-9]\+\.[0-9]\+\).*/\1/p')
+  local DOCKERAPIVERSION
+  DOCKERAPIVERSION=$(docker version | grep "API version" | head -1 | sed -ne 's/.*\([0-9]\+\.[0-9]\+\).*/\1/p')
+  doVersionCompare "1.3.0" "${DOCKERENGVERSION}"
+  local res1=$?
+  doVersionCompare "1.25" "${DOCKERAPIVERSION}"
+  local res2=$?
+  local ressum=$((res1 + res2))
+  doWarning "Please be aware, that if you want to use docker-compose with lazydocker, then you'll need at least version 1.23.2 of docker-compose"
```

Comment:
So this WARNING is printed every time regardless if the versions are fine or not?

Reply:
Now there is a separate if condition for the case that docker-compose is at an insufficient version or not running. So users with the propper version won't be bothered by this warning anymore.